### PR TITLE
Enhance PowerShell documentation

### PR DESCRIPTION
**Pull request by jescalada**
This is another PR for enhancing the available documentation. I felt that the PowerShell guide could use some improvements to reduce friction for new users, so I simplified the installation and basic usage steps. This guide still assumes that the user is familiar with PowerShell scripting.  

Feel free to make any changes as required.

## Changes
- Clarified installation steps using nuget (for Windows users specifically)
- Added extra requirements for scripts to execute (editing versions, selecting architecture)
  - Required for `Copy-Item` snippets to work correctly
  - Added reminder for user to change the version/architecture
- Added basic powershell usage: loading `ParquetSharp.dll` and getting a reference to a `ParquetFileReader`
  - Lists the previous example script if user wants more advanced functionality

### CODE REVIEW

File: docs/PowerShell.md [hunk 1]
```diff
@@ -1,10 +1,56 @@
...
 It's possible to use ParquetSharp from PowerShell.
-You can install ParquetSharp with the [NuGet command line interface](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference),
-then use `Add-Type` to load `ParquetSharp.dll`.
-However, you must ensure that the appropriate `ParquetSharpNative.dll` for your architecture and OS can be loaded as required,
-either by putting it somewhere in your `PATH` or in the same directory as `ParquetSharp.dll`.
-For examples of how to use ParquetSharp from PowerShell,
-see [these scripts from Apteco](https://github.com/Apteco/HelperScripts/tree/master/scripts/parquet).
+You can install ParquetSharp with the [NuGet command line interface](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference).  
+
+First, make sure `nuget.exe` is in your `PATH`, or in the current directory, then run the following to install the latest version of ParquetSharp into a new directory called `lib`:
```
Comment (adamreeve) [inline docs/PowerShell.md:8]: These instructions are presented as if this is the one way of using ParquetSharp from PowerShell, but there's no requirement that the dlls are in a directory named "bin" for example. I think it's worth clarifying that this is just one possible approach, and we should keep the existing text that says that the main requirement is that `ParquetSharpNative.dll` needs to be on the `PATH` or in the same directory as `ParquetSharp.dll`.

File: docs/PowerShell.md [hunk 2]
```diff
@@ -1,10 +1,56 @@
...
+
+The available runtime architectures are `win-x64`, `linux-x64`, `linux-arm64`, `osx-x64`, and `osx-arm64`.
+
+### Usage
+Use `Add-Type` to load `ParquetSharp.dll`:
+
+```powershell
+$dlls = Get-ChildItem -Path ".\bin" -Filter "*.dll" | where { @("ParquetSharpNative.dll") -notcontains $_.Name }
+
+$dlls | ForEach {
```
Comment (adamreeve) [inline docs/PowerShell.md:43]: Is it necessary to explicitly load each dll from the bin directory? I would expect that you can just do `Add-Type -Path bin\ParquetSharp.dll` and dlls that are dependencies will be loaded as required?

Eg. this worked for me when I originally looked into this: https://github.com/G-Research/ParquetSharp/issues/239#issuecomment-995191713

Comment (jescalada) [reply docs/PowerShell.md:44]: I replaced some of the existing code with the one you detailed in the PR. I think it's a lot easier to understand. 🤔

**Review by adamreeve — APPROVED** [review]
:+1: thanks @jescalada

**Pull request merged**